### PR TITLE
Add Guild XP, Tip, and Watchdog Message Blocking

### PR DESCRIPTION
**Pull request by Moire9**
This will allow you to block the guild exp messages in Skyblock and The Pit (and more, as long as they use the same format as Skyblock). It also can block the [WATCHDOG ANNOUNCEMENTS] that report ban numbers, and tip messages (good if you use autotip).

### CODE REVIEW

**Comment by Moire9** [comment]
I didn't realize that Draft PR would make it public. I thought it would be a thing that would be saved to my profile and I could come back to it later. This PR is almost done, I just want to do some more testing on it. I apologize for the incomplete PR.

**Comment by Moire9** [comment]
I'm closing this pull request while investigating a potential memory leak within my fork. It will be reopened when I determine if it's from TC.

**Comment by Moire9** [comment]
Alright. All good, and ready to be merged.

**Comment by Moire9** [comment]
God dammit. It appears the Watchdog message is having issues, but I don't know why. And it's hard to test. Closing until I can confirm it works.

**Comment by Moire9** [comment]
Okay. It's finally ready, I believe. Due to shortcomings of AutoTip, I cannot remove the blank chat messages surrounding the Watchdog Announcements - doing so would require changing the message system to allow classes to request more than one message at once, and remove them all, which is not something that I will do (at least, not something I will do now, after midnight). If it is alright with you, I will do my best to add such features into AutoTip tomorrow, otherwise, we can leave it as-is.

File: src/main/java/me/boomboompower/togglechat/toggles/defaults/TypeGexp.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/*
+ *     Copyright (C) 2020 SirNapkin1334
```
Comment (boomboompower) [inline src/main/java/me/boomboompower/togglechat/toggles/defaults/TypeGexp.java:2]: Are you fine with me adding Isophene on the side like `Isophene, SirNapkin1334`? It's completely up to you. Same with a few of the other files

**Review by boomboompower — COMMENTED** [review]
I'll approve this once we work out what you'd like to do with the licence. I don't really mind too much

**Comment by boomboompower** [comment]
I've attributed your account to the files you added as a CODEOWNER (as referenced in the above commit). 

As stated; once you let me know what you'd like to do with the licence I'll give it the all clear. 

CI is passing and you use compiled patterns so performance shouldn't be impacted by these changes. It looks good to me

File: src/main/java/me/boomboompower/togglechat/toggles/defaults/TypeGexp.java (hunk 1 above)
Comment (Moire9) [reply src/main/java/me/boomboompower/togglechat/toggles/defaults/TypeGexp.java:2]: Yes, that's fine for all of them.

**Comment by Moire9** [comment]
Yeah, that's fine, considering I used his code as a basis for my code. Out of curiosity though, who is Isophene? Why does boomboompower/ToggleChat redirect to Isophene/ToggleChat? And why for the copyright, Ipsophene & SirNapkin1334, opposed to boomboompower & SirNapkin1334? Considering all the other files I looked at said copyright boomboompower.

**Comment by boomboompower** [comment]
> Yeah, that's fine, considering I used his code as a basis for my code. Out of curiosity though, who is Isophene? Why does boomboompower/ToggleChat redirect to Isophene/ToggleChat? And why for the copyright, Ipsophene & SirNapkin1334, opposed to boomboompower & SirNapkin1334? Considering all the other files I looked at said copyright boomboompower.

Isophene is my startup, I own it. They are the old Copyright texts 😅
As for why they boomboompower/ToggleChat redirects, I used to host it there but I moved it to this organization for better management.

**Pull request merged**